browser_history/generic.py

```diff
@@ -200,6 +201,7 @@ def __init__(self):
         # format map is used by the formatted method to call the right formatter
         self._format_map = {
             'csv': self.to_csv,
+            'json': self.to_json
```

Comment:
Doing this does not allow the CLI to get plain JSON output. You could use `json` for plain JSON (maybe with a lambda function, not sure about this) and `jsonl` for JSON lines.

Reply:
actually I made a function to_json and added a parameter json_lines as bool to switch between JSON lines and plain JSON by default its JSON lines. Yes, I can add a separate format map for each plain JSON and JSON lines that would be better.